Framework/PythonInterface/mantid/utils/AbsorptionCorrUtils.py

```diff
@@ -0,0 +1,284 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2018 ISIS Rutherford Appleton Laboratory UKRI,
```

Comment:
One last thing - is it worth updating the year as this is a new file?

Reply:
Could I be slightly annoying and ask we rename the file to `absorptioncorrutils.py` for a more Pythonic naming convention inline with the rest of the `mantid` module?